docs/reference/client-side-encryption.txt

```diff
@@ -412,6 +420,10 @@ for your master key. You will use that information to generate a data key.
         tenant_id: 'AZURE-TENANT-ID',
         client_id: 'AZURE-CLIENT-ID',
         client_secret: 'AZURE-CLIENT-SECRET'
+      },
+      gcp: {
+        email: 'GCP-EMAIL',
+        private_key: 'GCP-PRIVATE-KEY',
```

Comment:
I would specify what format the private key can be in here, in light of the test code wrapping it with pem line markers. Is it some format that GCP provides?

Reply:
We managed to accept both DER and PEM. Documentation updated accordingly.